src/appleseed/foundation/image/colormap.h

```diff
@@ -93,6 +96,10 @@ class ColorMap
 
     Color3f evaluate_palette(float x) const;
 
+    APPLESEED_DLLSYMBOL std::array<float,4> compute_tile_highlight_color(
+        const size_t thread_index,
```

Comment:
If we rename the method, those should be renamed as well (`index`, `count` sound good enough).